geojson2svg/svg2png.go

```diff
@@ -0,0 +1,126 @@
+package geojson2svg
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/ONSdigital/go-ns/log"
+)
+
+const (
+	// ArgSVGFilename is text that will be replaced with the actual svg filename when invoking the PNGConverter executable
+	ArgSVGFilename = "<SVG>"
+	// ArgPNGFilename is text that will be replaced with name of the png file to write when invoking the PNGConverter executable
+	ArgPNGFilename = "<PNG>"
+)
+
+// PNGConverter invokes an executable file to convert an svg file to png
+type executablePNGConverter struct {
+	Executable string
+	Arguments  []string
+}
+
+// NewPNGConverter creates a new PNGConverter that invokes an executable to perform the conversion.
+// Parameters:
+// executable - the path to the executable that converts an svg to png.
+// arguments - the arguments passed to the executable. These should include:
+// 		geojson2svg.ArgSVGFilename as the name of the svg file to convert
+// 		geojson2svg.ArgPNGFilename as the name of the png file to create
+func NewPNGConverter(executable string, arguments []string) PNGConverter {
+	return &executablePNGConverter{Executable: executable, Arguments: arguments}
+}
+
+// Convert converts the given svg file to a base64-encoded png
+func (exe *executablePNGConverter) Convert(svg []byte) ([]byte, error) {
+
+	tempName := "temp_" + randomString(8)
```

Comment:
I know these get cleaned up on completion but it might be useful to log the name of the temp files  and reference them in the log statements.